hathor/p2p/manager.py

```diff
@@ -106,6 +107,9 @@ def __init__(self, reactor: ReactorBase, my_peer: PeerId, server_factory: 'Hatho
 
         self.ssl = ssl
 
+        # Parameter to explicitly enable whitelist-only mode, when False it will still check the whitelist for sync-v1
+        self.whitelist_only = whitelist_only
```

Comment:
Shouldn't we use the netfilter to do it?

Reply:
I think it would be great. But maybe on a separate PR? I don't think it will be a trivial change.

Reply:
Sure. And we are not using it here, so it seems safe to just remove it. What do you think?

Reply:
I agree. Thanks!